src/abstract-classes/browser-plugin.ts

```diff
@@ -25,7 +25,7 @@ export interface CommonLibrary {
  * @internal
  */
 export interface CommonBrowser {
-    newPage(...args: unknown[]): unknown;
+    newPage(...args: unknown[]): any;
```

Comment:
Why the change?

Reply:
Tried making an interface for it too, but I'd always get errors from WeakMap, and it doesn't really matter since this is overwritten by generics everywhere
![image](https://user-images.githubusercontent.com/17960496/129049332-ce0a7b56-bebe-449f-8fef-b8bce68467cf.png)

Reply:
It do be like that sometimes. Glad this is now resolved :)